app/javascript/mastodon/reducers/timelines.js

```diff
@@ -36,7 +36,7 @@ const initialTimeline = ImmutableMap({
   items: ImmutableList(),
 });
 
-const isPlaceholder = value => value === TIMELINE_GAP || value === TIMELINE_SUGGESTIONS;
+const isNonStatusId = value => !Number.isInteger(value);
```

Comment:
But status identifiers are strings?

Reply:
ah whoops, for some reason i thought that function parsed strings correctly. i replaced it with a `parseInt` + `isNaN` check, which can still technically pass negative numbers but i don't expect that to happen.

Reply:
Yeah that should work but as I wrote below, I'm not sure we want to do it like this. The numbers are too big to be stored as integers (which is fine, it doesn't break the logic), but also our API doesn't *require* them to be numbers-as-string. This is not an issue for our project itself but other projects use our UI and may use it with different ID schemes (e.g. iirc there are people running forks of the Mastodon front-end with Akkoma).

Additionally, if a special value starts with a number but continues on with letters, `isNonStatusId` will return false.

So I'd prefer an explicit list of our special values, but I understand this too has its issues, since we can forget updating the list.